man/IO_sink_ndjson.Rd

```diff
@@ -0,0 +1,61 @@
+% Generated by roxygen2: do not edit by hand
+% Please edit documentation in R/lazyframe__lazy.R
+\name{LazyFrame_sink_ndjson}
+\alias{LazyFrame_sink_ndjson}
+\title{Stream the output of a query to a JSON file}
+\usage{
+LazyFrame_sink_ndjson(
+  path,
+  maintain_order = TRUE,
+  type_coercion = TRUE,
+  predicate_pushdown = TRUE,
+  projection_pushdown = TRUE,
+  simplify_expression = TRUE,
+  slice_pushdown = TRUE,
+  no_optimization = FALSE,
+  inherit_optimization = FALSE
+)
+}
+\arguments{
+\item{path}{File path to which the result should be written.}
+
+\item{maintain_order}{Keep the same order as the original \code{DataFrame}. Setting
```

Comment:
Is this explanation appropriate for LazyFrame's methods?

Reply:
It was the same problem for other sink functions, I harmonized it to have "Lazyframe" instead